script/deploy/healthCheck.ts

```diff
@@ -416,33 +416,7 @@ const main = defineCommand({
       //          ╭─────────────────────────────────────────────────────────╮
       //          │                Check emergency pause config             │
       //          ╰─────────────────────────────────────────────────────────╯
-      consola.box('Checking emergency pause config...')
```

Comment:
We don't want to check this anymore?

Reply:
Because we moved RPC URLS into MongoDB and they are not pulled from git secrets anymore